src/WebApiGateway/Configuration/FilterConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using WebApiGateway.Filters;
+using WebApiGateway.Services.Contracts;
+using WebApiGateway.Services.Implementations;
+
+namespace WebApiGateway.Configuration
+{
+    public static class FilterConfiguration
+    {
+        public static IServiceCollection AddFilterConfiguration(this IServiceCollection services)
+        {
```

Comment:
Move services.AddScoped<IAuthClaimService, AuthClaimService>(); to services di extension